std_blocks/trig/trig.c

```diff
@@ -127,12 +128,12 @@ int do_trigger(struct trig_inf *inf)
 /* step */
 void trig_step(ubx_block_t *b)
 {
-	DBG(" ");
+	ubx_debug(b, " ");
```

Comment:
the empty DBG served as a "tracepoint", because it included __FUNCTION__, but ubx_debug doesn't and it's ugly anyway, so I suggest to remove it.